code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -380,3 +380,13 @@
 	New()
 		..()
 		reagents.add_reagent("frostoil", 30)
+
+/obj/item/weapon/reagent_containers/glass/bottle/antisocial
+	//No special name or description
+	name = "bottle"
+	desc = "A small bottle."
+	icon = 'icons/obj/chemical.dmi'
+

```

Comment:
Before anyone else comments on it, yes, these are all three variables that get inherited from the parent as exactly that anyway. Yes, this is a waste of three lines. I'll look at it next commit.